models/target.go

```diff
@@ -19,62 +19,75 @@ package models
 
 import (
 	"github.com/greenbone/eulabeia/messages"
+	"github.com/greenbone/eulabeia/messages/cmds"
 	"github.com/greenbone/eulabeia/messages/info"
 )
 
 // References are the bid, cve or xrefs entries in a nasl script.
 //eg. an xref from nmap.nasl is type 'URL' with id: 'https://nmap.org/book/man-performance.html'
 type RefType struct {
-	Type string
-	ID   string
+	Type string `json:"type"`
+	ID   string `json:"id"`
 }
 
 // VT's parameters have an ID. The ID 0 (zero) is reserved for the script timeout which is not add to Redis cache as a preference but as script parameter.
 type VTParamType struct {
-	ParameterID           int    // Parameter ID. ID:0 is always to specify a timeout.
-	ParameterName         string // Parameter Name
-	ParameterValue        string // Parameter Value
-	ParameterType         string // Parameter Type
-	ParameterDescription  string // Parameter description
-	ParameterDefaultValue string // Parameter default value
+	ParameterID           int    `json:"id"`          // Parameter ID. ID:0 is always to specify a timeout.
+	ParameterName         string `json:"name"`        // Parameter Name
+	ParameterValue        string `json:"value"`       // Parameter Value
+	ParameterType         string `json:"type"`        // Parameter Type
+	ParameterDescription  string `json:"description"` // Parameter description
+	ParameterDefaultValue string `json:"default"`     // Parameter default value
 }
 
 // Severities are stored as Tag. Because tag names can not be repeated, only one severity is supported.
 // The old tag name for severities are cvss_base and cvss_base_vector. The new extended severities tags has priority over the old format.
 type SeverityType struct {
-	SeverityVector  string // CVSS vector. Supported are CVSSv2 and CVSSv3.x
-	SeverityVersion string // CVSS version
-	SeverityDate    string // CVE creation date. Default to VT creation date
-	SeverityOrigin  string // Serverity Origin
+	SeverityVector  string `json:"severity_vector"` // CVSS vector. Supported are CVSSv2 and CVSSv3.x
```

Comment:
Why do the field names have the prefix Severity* when the struct name is already `SeverityType` and the key in json within VT is also called `severity`?

Reply:
I just took it as it was. I change it also for Parameter.